game/scripts/vscripts/heroes/vengefulspirit/command_aura.lua

```diff
@@ -0,0 +1,165 @@
+---@class vengefulspirit_command_aura_lua : CDOTA_Ability_Lua
+vengefulspirit_command_aura_lua = class({})
+
+---@class modifier_vengefulspirit_command_aura_lua : CDOTA_Modifier_Lua
+modifier_vengefulspirit_command_aura_lua = class({})
+LinkLuaModifier( "modifier_vengefulspirit_command_aura_lua", LUA_MODIFIER_MOTION_NONE )
+
+---@class modifier_vengefulspirit_command_aura_effect_lua : CDOTA_Modifier_Lua
+modifier_vengefulspirit_command_aura_effect_lua = class({})
+LinkLuaModifier( "modifier_vengefulspirit_command_aura_effect_lua", LUA_MODIFIER_MOTION_NONE )
+
+---@return string
+function vengefulspirit_command_aura_lua:GetIntrinsicModifierName()
+	return "modifier_vengefulspirit_command_aura_lua"
+end
+
+---@return boolean
+function modifier_vengefulspirit_command_aura_lua:IsHidden()
+    return true
+end
+
+---@return boolean
+function modifier_vengefulspirit_command_aura_lua:IsAura()
+    return true
+end
+
+---@return string
+function modifier_vengefulspirit_command_aura_lua:GetModifierAura()
+    return "modifier_vengefulspirit_command_aura_effect_lua"
+end
+
+---@return DOTA_UNIT_TARGET_TEAM
+function modifier_vengefulspirit_command_aura_lua:GetAuraSearchTeam()
+    return DOTA_UNIT_TARGET_TEAM_FRIENDLY
+end
+
+---@return DOTA_UNIT_TARGET_TYPE
+function modifier_vengefulspirit_command_aura_lua:GetAuraSearchType()
+    return DOTA_UNIT_TARGET_HERO + DOTA_UNIT_TARGET_CREEP
+end
+
+---@return DOTA_UNIT_TARGET_FLAGS
+function modifier_vengefulspirit_command_aura_lua:GetAuraSearchFlags()
+    return DOTA_UNIT_TARGET_FLAG_INVULNERABLE
+end
+
+---@return number
+function modifier_vengefulspirit_command_aura_lua:GetAuraRadius()
+    return self.aura_radius
+end
+
+function modifier_vengefulspirit_command_aura_lua:OnCreated( kv )
+    self.aura_radius = self:GetAbility():GetSpecialValueFor( "aura_radius" )
+    if IsServer() and self:GetParent() ~= self:GetCaster() then
+        self:StartIntervalThink( 0.5 )
+    end
+end
+
+function modifier_vengefulspirit_command_aura_lua:OnRefresh( kv )
+    self.aura_radius = self:GetAbility():GetSpecialValueFor( "aura_radius" )
+end
+
+---@return table
+function modifier_vengefulspirit_command_aura_lua:DeclareFunctions()
+    local funcs = {
+        MODIFIER_EVENT_ON_DEATH
+    }
+    return funcs
+end
+
+---@class OnDeathParams
+---@field attacker CDOTA_BaseNPC
+---@field unit CDOTA_BaseNPC
+
+---@param params OnDeathParams
+---@return number
+function modifier_vengefulspirit_command_aura_lua:OnDeath( params )
```

Comment:
This should be ---@override, need to find a way to move the OnDeathParams and doc to moddota lua docs.